pybliometrics/scopus/abstract_retrieval.py

```diff
@@ -551,12 +551,13 @@ def website(self):
 
     def __init__(self, identifier=None, refresh=False, view='META_ABS',
                  id_type=None):
-        """Interaction with the Abstract Retrieval API.
+        """Class to represent the results from retrieval request from the
+        Scopus Abstract API.
```

Comment:
The deleted line is the more recent one. I guess something went wrong when rewinding the fork.